src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/UseLiteralsWhereAppropriate.cs

```diff
@@ -61,6 +61,7 @@ public override void Initialize(AnalysisContext analysisContext)
                 var lastField = fieldInitializer?.InitializedFields.LastOrDefault();
                 var fieldInitializerValue = fieldInitializer?.Value;
                 if (fieldInitializerValue == null ||
+                    lastField == null ||
```

Comment:
💭 Could this one have been a bug? Maybe try to add a test to hit this case?

Reply:
Avoiding adding unit tests in this PR. I will leave this comment unresolved and file a followup bug to track any such cases.